kaggle_environments/envs/connectx/connectx.json

```diff
@@ -23,12 +23,12 @@
       "default": 4,
       "minimum": 1
     },
-    "agentTimeout": 16,
-    "actTimeout": 8,
+    "agentTimeout": 60,
```

Comment:
Several of the environment default configurations were out of date with production, I've updated all of them and added a corresponding observation.remainingOverageTime field that matches each agentTimeout field.